src/machine.rs

```diff
@@ -33,6 +33,7 @@ use crate::{
 
 // Some global facts about the emulated machine.
 pub const PAGE_SIZE: u64 = 4 * 1024; // FIXME: adjust to target architecture
+pub const SIGRTMAX: i32 = 42; // should be between 32 and 64
```

Comment:
Please explain what the constant means, or better give it a more descriptive name. SIGRTMAX looks like a randomly generated password. ;)